### PR TITLE
fix change vxnet issue

### PR DESCRIPTION
**Pull request by hlwanghl**
Signed-off-by: Hongliang Wang <hongliangwang@yunify.com>

### CODE REVIEW

**Comment by hlwanghl** [comment]
Fix issues when change from VPC1 (192.168.0.0) to VPC2 (192.168.0.0):

Node | Source | Target | Result
-|-|-|-
1/cln-xxxxxxx1/ | 192.168.0.2 | 192.168.0.3 | 192.168.0.2
2/cln-xxxxxxx2/ | 192.168.0.3 | 192.168.0.4 | 192.168.0.2
3/cln-xxxxxxx3/ | 192.168.0.4 | 192.168.0.2 | 192.168.0.2

The final "master" will be `192.168.0.2`, and sentinel.conf will be corrupted.

File: ansible/roles/redis-standalone/files/etc/confd/templates/redis-server.sh/04.hosts.tmpl [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+sed "/^# >> Redis nodes./,/^# << Redis nodes./d" $hostsFile > $hostsFile.swap
+
+flush >> $hostsFile.swap << HOSTS_FILE
+# >> Redis nodes. WARNING: this is managed by script and please don't touch manually.
+{{- range lsdir "/hosts" }}
+{{ getv (printf "/hosts/%s/ip" .) }}{{ printf "\t" }}{{ getv (printf "/hosts/%s/node_id" .) }}
+{{- end }}
+# << Redis nodes. WARNING: this is managed by script and please don't touch manually.
+HOSTS_FILE
+mv $hostsFile.swap $hostsFile
```
Comment (MrYourM) [inline ansible/roles/redis-standalone/files/etc/confd/templates/redis-server.sh/04.hosts.tmpl:11]: 该文件看起来没什么用

Comment (hlwanghl) [reply ansible/roles/redis-standalone/files/etc/confd/templates/redis-server.sh/04.hosts.tmpl]: 好的，已删。

**Review by MrYourM — APPROVED** [review]
/lgtm

**Pull request merged**